.github/maven/synoki.xml

```diff
@@ -2,46 +2,12 @@
 
   <servers>
     <server>
-      <id>synoki-nexus</id>
+      <id>ossrh</id>
       <username>${env.MAVEN_USER}</username>
       <password>${env.MAVEN_PASSWORD}</password>
```

Comment:
Do we still need this file at all? If we do, can you rename it to `settings.xml`?

Reply:
Renamed